crates/biome_js_analyze/src/lint/nursery/use_sorted_classes/tailwind_preset.rs

```diff
@@ -572,11 +575,189 @@ const UTILITIES_LAYER_CLASSES: [&str; 569] = [
     "duration-",
     "ease-",
     "will-change-",
+    "contain-none$",
+    "contain-content$",
+    "contain-strict$",
+    "contain-size$",
+    "contain-inline-size$",
+    "contain-layout$",
+    "contain-paint$",
+    "contain-style$",
     "content-",
     "forced-color-adjust-auto$",
     "forced-color-adjust-none$",
 ];
 
+lazy_static! {
+    pub static ref VARIANT_CLASSES: [&'static str; 165] = [
```

Comment:
Arrays don't need to be initialized with lazy static. If you ended using lazy static, it means that there's a refactor opportunity. This could be `pub const` like other arrays

Reply:
> Arrays don't need to be initialized with lazy static. If you ended using lazy static, it means that there's a refactor opportunity. This could be `pub const` like other arrays

Facepalmed myself after realizing that. Thank you for pointing that out!